src/Controller/VehicleController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\VehicleMake;
+use App\Entity\VehicleModel;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+
+class VehicleController extends AbstractController
+{
+    /**
+     * @Route("/vehicle", name="vehicleMakes", methods={"GET"})
+     */
+    public function getVehicleMakes()
+    {
+        $makes = $this->getDoctrine()
+            ->getRepository(VehicleMake::class)
+            ->findAll();
+        if (!$makes) {
+            $response = new JsonResponse(['error' => 'No makes could be found.']);
+
+            $response->setStatusCode(404);
+            return $response;
```

Comment:
Galejai viena eilute ta pati padaryt `return $this->json(['error` => '...'], 404);`